src/features/ui/theme/typography.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from 'styled-components'
+
+import { colors } from './colors'
+import { mq } from './mq'
+
+export const font = {
+  base: '"Hind", sans-serif',
+  headings: '"Hind", sans-serif',
+  quotes: '"Playfair Display", sans-serif',
+}
+
+const allHeadingsStyle = css`
+  font-family: ${font.headings};
+  font-weight: inherit;
+`
+
+const allLabelsStyle = css`
+  font-family: ${font.base};
+  font-weight: inherit;
+  text-transform: uppercase;
+  letter-spacing: 0.08em;
+  font-weight: 600;
```

Comment:
Is it intentional to have two `font-weight` props in a row here ? 🤔

Reply:
Good catch sir! I don't believe it is. @Hofik9 confirm?